### PR TITLE
create blog and header component

### PR DESCRIPTION
**Pull request by shouryabansal7**
created blog and header component

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,3 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```
Comment (Kushagraw12) [inline src/index.css:3]: This was the font for the whole UI, right @neuralaidtu , why did you remove this?

**Comment by shouryabansal7** [comment]
It was already not added as index.css file was not imported we can add it
again
